Jenkinsfile

```diff
@@ -53,12 +59,12 @@ node('linux && docker') {
     }
 
     withDockerContainer(image: 'node:14', args: '-u=root') {
-      stage('Bump version') {
+      stage('Commit bumped version') {
         withCredentials([
         usernameColonPassword(credentialsId: 'github-commit-token', variable: 'GH_CREDENTIALS')]) {
           sh 'npm run bump:version:pre'
         }
-     }
+      }
```

Comment:
The version is bumped again here, since the files modified by the previous bump are [cleaned beforehand](https://github.com/coveo/ui-kit/pull/633/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R56).